great_expectations/core/configuration.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, id_: Optional[str] = None, name: Optional[str] = None) -> Non
         # Note: name and id are optional currently to avoid updating all documentation within
         # the scope of this work.
         if id_ is not None:
-            self.id = id_
+            self.id_ = id_
```

Comment:
Should this be reflected consistently throughout? I think we may need to change RBP.

Reply:
Yes! I did that in a later commit: https://github.com/great-expectations/great_expectations/pull/5591/commits/419d9e8e7376123c426932cf680b421068dc0764